apps/lobe-chat/1.35.12/docker-compose.yml

```diff
@@ -1,6 +1,6 @@
 services:
   lobe-chat:
-    image: lobehub/lobe-chat:v1.35.12
+    image: lobehub/lobe-chat:v1.35.13
     container_name: ${CONTAINER_NAME}
     restart: always
     networks:
```

Comment:
It seems like you've included some code but forgot to include "## Check here if there are any discrepancies or errors that need to be addressed ##", which is necessary to provide more detailed feedback on the differences. Please ensure that all relevant sections of the code are provided so I can offer guidance on any inconsistencies or improvements needed.